examples/all-clusters-app/nrfconnect/child_image/mcuboot/prj_release.conf

```diff
@@ -28,3 +28,6 @@ CONFIG_CONSOLE=n
 CONFIG_SERIAL=n
 CONFIG_UART_CONSOLE=n
 CONFIG_USE_SEGGER_RTT=n
+
+# Use minimal C library instead of the Picolib
+CONFIG_MINIMAL_LIBC=y
```

Comment:
I wonder... could this be set `Kconfig.mcuboot.defaults`, instead?

Reply:
Yes, done. I think we did it in prj.confs before, as it was a quick fix without modifying the connectedhomeip fork.